src/components/benefits/benefits.js

```diff
@@ -12,7 +12,7 @@ import {
 
 const Benefits = () => {
   return (
-    <StyledBenefits>
+    <StyledBenefits id="benefits">
```

Comment:
why is the benefit section hiding the section header when it scrolls to it? i think the header makes it hidden.